www/src/info.md

```diff
@@ -43,6 +43,20 @@ The first login sets your password.
 - You can store up to [{{ config.max_mailbox_size }} messages on the server](https://delta.chat/en/help#what-happens-if-i-turn-on-delete-old-messages-from-server).
 
 
+### <a name="account-deletion"></a> Account deletion 
+
+If you remove a {{ config.mail_domain }} profile from within the Delta Chat app, 
+then the according account on the server, along with all associated data,
+is automatically deleted {{ config.delete_inactive_users_after }} days afterwards. 
+
+If you use multiple devices 
+then you need to remove the according chat profile from each device
+in order for all account data to be removed on the server side. 
+
+If you have any further questions or requests regarding account deletion
+please send a message from your account to {{ config.privacy_mail }}. 
```

Comment:
If `privacy_mail` is unset it will look weird, OTOH chatmail admins really should have a privacy policy and can just remove the sentence otherwise.

Reply:
But we could make an if clause around the paragraph.

Reply:
Maybe it's better to show a broken message than just removing the sentence.  This way the user can become aware that they don't have any handle on who runs the server ;)